docs/installation.rst

```diff
@@ -56,15 +58,15 @@ Execute the following command to install the latest stable version of Transforme
 
   pip3 install --no-build-isolation git+https://github.com/NVIDIA/TransformerEngine.git@stable
 
-This will automatically detect if any supported deep learning frameworks are installed and build Transformer Engine support for them. To explicitly specify frameworks, set the environment variable `NVTE_FRAMEWORK` to a comma-separated list (e.g. `NVTE_FRAMEWORK=jax,pytorch`).
+This will automatically detect if any supported deep learning frameworks are installed and build Transformer Engine support for them. To explicitly specify frameworks, set the environment variable `NVTE_FRAMEWORK` to a comma-separated list (for example, `NVTE_FRAMEWORK=jax,pytorch`).
 
 Installation (development build)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. warning::
 
    While the development build of Transformer Engine could contain new features not available in
-   the official build yet, it is not supported and so its usage is not recommended for general
+   the official build, those features are not supported, so their usage is not recommended for general
```

Comment:
We meant to say here that the development build of Transformer Engine is not supported, not that those features are not supported.

Reply:
:+1: